instances/technique/imageRegistration.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/registration",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "An 'image registration' is a process of bringing a set of images into the same coordinate system.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "transformation",
```

Comment:
`@id` vs. `name`: registration or transformation? Definition points to registration, but also says "image registration" which is yet another thing.

Reply:
screw up on my side... should be corrected now